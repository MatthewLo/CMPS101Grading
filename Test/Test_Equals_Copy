5
Check list equals and copy
gcc -std=c99 -Wall -c Test_Equals_Copy.c
gcc -std=c99 -Wall -o Test_Equals_Copy Test_Equals_Copy.o ../List.o
./Test_Equals_Copy
valgrind --leak-check=full --log-file="memoryLeak" ./Test_Equals_Copy
