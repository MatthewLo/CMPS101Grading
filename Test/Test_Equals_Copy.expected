0 1 4 9 16 
0 1 4 9 16 
0
1
1
